STACK-UP-3/warriorz-backend
# #173327425 add fixing accommodation

#### What does this PR do?
This PR fixes an issue with the ability of a requester to view individual accommodation.

#### Description of Task to be completed?
- add the ability for a requester to view a specific accommodation

#### How should this be manually tested?
> As a requester will try to view an individual accommodation he/she should be able to view it well

#### Any background context you want to provide?
Initially, a requester was not able to view a specific accommodation.

#### What are the relevant pivotal tracker stories?
[#173327425](https://www.pivotaltracker.com/story/show/173327425)

#### Screenshots
N/A
